sass/main.scss

```diff
@@ -0,0 +1,5 @@
+@import "base";
+@import "typography";
```

Comment:
Please review this sass "import" if you are not using it kindly removed it to had a great presentation.